.github/workflows/security.yml

```diff
@@ -110,8 +133,26 @@ jobs:
       - name: Flag snap scanning alerts
         run: |
           cd /home/runner
-          jq '.runs[].tool.driver.rules[] |= (.shortDescription.text |= "Snap scan - " + .)' ${{ matrix.version }}-stable.sarif > tmp.json
-          mv tmp.json ${{ matrix.version }}-stable.sarif
+          cat <<< $(jq '.runs[].tool.driver.rules[] |= (.shortDescription.text |= "Snap scan - " + .)' ${{ matrix.version }}-stable.sarif) > ${{ matrix.version }}-stable.sarif
```

Comment:
@kadinsayani please can you add a comment as to what this is doing, its not that clear.